crates/client/gateway/src/client/builder.rs

```diff
@@ -141,7 +141,7 @@ impl retry::Policy<Request<String>, Response<Incoming>, Box<dyn Error + Send + S
                     let next_policy = self.clone();
                     let fut = async move {
                         if (*pause_until.read().await).is_none() {
-                            log::info!("The fetching process has been rate limited, retrying");
+                            tracing::info!("The fetching process has been rate limited, retrying");
```

Comment:
Can we instrument train implementation? Would be nice to print this inside a span.